### DISCUSSION TITLE
Socket io client not setting header

### DISCUSSION THREAD

**Discussion by Gfhandel** [socketio/socket.io#5247] [Q&A]
My project backend uses header authorization to authenticate users. 

Socket io client is what I use on the client side, but it does not set nor send headers.

**Comment by Bubuioc** [comment]
Hello, if you mean by headers JWT, the problem can be in your transports settings. if you enable only websockets or put websockets on the first place and your client is from browser you cannot send extraheaders, check out this link [here](https://socket.io/how-to/use-with-jwt).There is a red card check info from this.
